README.md

```diff
@@ -100,6 +100,21 @@ module.exports = {
 }
 ```
 
+If you use `webpack 2.x`, you have to pass options object to `eslint-loader` by using `LoaderOptionsPlugin` ([webpack 2/docs](https://webpack.js.org/guides/migrating/#loaderoptionsplugin-context)).
```

Comment:
This is not supposed to be necessary. eslint-loader already support options from ``query``. Can you double check and remove this?